docs/content/routing/entrypoints.md

```diff
@@ -51,199 +56,414 @@ They define the port which will receive the requests (whether HTTP or TCP).
+            requestAcceptGraceTimeout = 42
+    ```
+    
+    ```yaml tab="File (YAML)"
+    ## Static configuration
+    entryPoints:
+      name:
+        address: ":8888"
+        transport:
+          lifeCycle:
+            requestAcceptGraceTimeout: 42
+    ```
+    
+    ```bash tab="CLI"
+    ## Static configuration
+    --entryPoints.name.address=:8888
+    --entryPoints.name.transport.lifeCycle.requestAcceptGraceTimeout=42
+    ```
 
+??? info "`lifeCycle.graceTimeOut`"
+    
+    _Optional, Default=10s_
+    
+    Duration to give active requests a chance to finish before Traefik stops.
+    
+    Can be provided in a format supported by [time.ParseDuration](https://golang.org/pkg/time/#ParseDuration) or as raw values (digits).
+    
+    If no units are provided, the value is parsed assuming seconds.
+    
+    !!! warning "In this time frame no new requests are accepted."
+    
     ```toml tab="File (TOML)"
+    ## Static configuration
+    [entryPoints]
+      [entryPoints.name]
+        address = ":8888"
+        [entryPoints.name.transport]
+          [entryPoints.name.transport.lifeCycle]
+            graceTimeOut = 42
+    ```
+    
+    ```yaml tab="File (YAML)"
+    ## Static configuration
+    entryPoints:
+      name:
+        address: ":8888"
+        transport:
+          lifeCycle:
+            graceTimeOut: 42
+    ```
+    
+    ```bash tab="CLI"
+    ## Static configuration
+    --entryPoints.name.address=:8888
+    --entryPoints.name.transport.lifeCycle.graceTimeOut=42
+    ```
+
+### ProxyProtocol
+
+Traefik supports [ProxyProtocol](https://www.haproxy.org/download/2.0/doc/proxy-protocol.txt) version 1 and 2.
+
+If proxyprotocol header parsing is enabled for the entry point, this entry point can accept connections with or without proxyprotocol headers.
+
+If the proxyprotocol header is passed, then the version is determined automatically.
+
+??? info "`proxyProtocol.trustedIPs`" 
+    
+    Enabling Proxy Protocol with Trusted IPs.
+
+    ```toml tab="File (TOML)"
+    ## Static configuration
     [entryPoints]
       [entryPoints.web]
         address = ":80"
     
-        [entryPoints.web.forwardedHeaders]
+        [entryPoints.web.proxyProtocol]
           trustedIPs = ["127.0.0.1/32", "192.168.1.7"]
     ```
     
     ```yaml tab="File (YAML)"
+    ## Static configuration
     entryPoints:
       web:
         address: ":80"
-        forwardedHeaders:
+        proxyProtocol:
           trustedIPs:
           - "127.0.0.1/32"
           - "192.168.1.7"
     ```
     
     ```bash tab="CLI"
     --entryPoints.web.address=:80
-    --entryPoints.web.forwardedHeaders.trustedIPs=127.0.0.1/32,192.168.1.7
+    --entryPoints.web.proxyProtocol.trustedIPs=127.0.0.1/32,192.168.1.7
     ```
 
-??? example "Insecure Mode -- Always Trusting Forwarded Headers"
+    IPs in `trustedIPs` only will lead to remote client address replacement: Declare load-balancer IPs or CIDR range here.
+
+??? info "`proxyProtocol.insecure`"
```

Comment:
```suggestion
??? info "`proxyProtocol.insecure`: Insecure Mode (Testing Environment Only)."
```

Reply:
I prefer to be homogeneous with other element.
The name of the option can be long.